src/DDTrace/Span.php

```diff
@@ -32,6 +32,8 @@ class Span extends DataSpan
         Tag::SERVICE_VERSION => true,
     ];
 
+    public $internalSpan;
```

Comment:
Is this to avoid a deprecation notice or something as per the PHP 8.2 change? Or something else?

Reply:
Somewhere else I need to access the ObjectKVStore::get() on the internalSpan.